File: 1_Year/AED l/www.beecrowd.com.br/Exercicios/triangulo_pascal.c
#include <stdio.h>
#include <math.h>

int main() {
    int t, n;
    scanf("%i", &t);
    
    while (t--) {
        scanf("%i", &n);
        printf("%i\n",(int) pow(2, n) - 1);
    }

    return 0;
}

File: 1_Year/CursoJava/triangulo_pascal.java
import java.util.Scanner;

public class triangulo_pascal {
    public static void main(String[] args) {
        double  n, s;
        Scanner scanf = new Scanner(System.in);
        int t = scanf.nextInt();
        
        while (t!=0) {
            n = scanf.nextInt();
            s= Math.pow(2, n) -1;
            System.out.printf("%.0f\n", s);   
            s=2;
            t--;
        }
        scanf.close();
    }
}
